### PR TITLE
Participant swap

### PR DESCRIPTION
**Pull request by swapnils118**
Fixes: #38 

## Description
- Added program information block component as per requirement.
- Added components to current /participants/:id route with layout foundation.
- Merge conflicts fixed from current working tree.
- Page title modified to Lung Health Foundation.

## Feature Link:
- Program Information component under participant details page.
- 

## Description:
Participant Information block added to /participant/:id route in sync with existing project structure and layout foundation given to overall page.
Browser's page title is also modified.

---

## Screenshots:
participant-details
---

## 📢 Notes for Reviewers
Add any additional information or context for the reviewers.

### CODE REVIEW

File: src/pages/participant-profile/index.tsx
Comment (oveed) [inline src/pages/participant-profile/index.tsx:1]: Well done trying to arrange them but keep the original performance measure component

**Review by oveed — CHANGES_REQUESTED** [review]
Everything else looks good

**Review by swapnils118 — COMMENTED** [review]
Approved

File: src/pages/participant-profile/index.tsx
Comment (swapnils118) [reply src/pages/participant-profile/index.tsx:1]: Fixed.
Can you please review it once.
Thanks

**Review by oveed — APPROVED** [review]

**Comment by HongyuLiuLance** [comment]
I see that you also made changes to the relevant files on the /staff_schedule page. Please upload screenshots or videos of all your implemented and changed parts so far(/staff_schedule page and participants/:id). Also, I don't know what is the purpose of the file src/pages/participants/index.tsx, if it is invalid comment code please remove it.

**Comment by swapnils118** [comment]
> I see that you also made changes to the relevant files on the /staff_schedule page. Please upload screenshots or videos of all your implemented and changed parts so far(/staff_schedule page and participants/:id). Also, I don't know what is the purpose of the file src/pages/participants/index.tsx, if it is invalid comment code please remove it.

**Comment by HongyuLiuLance** [comment]
> I see that you also made changes to the relevant files on the /staff_schedule page. Please upload screenshots or videos of all your implemented and changed parts so far(/staff_schedule page and participants/:id). Also, I don't know what is the purpose of the file src/pages/participants/index.tsx, if it is invalid comment code please remove it.

**Comment by swapnils118** [comment]
Hi @HongyuLiuLance, thanks for your review and feedback. I would be more than happy to resolve your concerns:
- I have optimized the code under staff-schedule for create new class modal which was my first contribution for this project and I will be adding ss of the modal and modified code for your reference enclosed within.
- About the other contributions, the respective PR's are already there waiting for approval.
- W.r.t participant/index.tsx, all the unused code is already removed in the latest PR.

Looking forward for any other query :)
modal
staff-schedule

**Comment by tianchengc** [comment]
@swapnils118 could you fix merge conflict, thanks

**Comment by swapnils118** [comment]
@tianchengc Yes I am working on it. Will be pushing the final commit soon.

**Pull request merged**